model_analyzer/entrypoint.py

```diff
@@ -121,6 +119,21 @@ def get_server_handle(config):
             log_path=config.triton_output_path,
             mounts=config.triton_docker_mounts,
             labels=config.triton_docker_labels)
+    elif config.triton_launch_mode == 'C_API':
+        triton_config = TritonServerConfig()
+        triton_config['model-repository'] = os.path.abspath(
+            config.output_model_repository_path)
+        logger.info("Starting a Triton Server using perf_analyzer's C_API...")
+        server = TritonServerFactory.create_server_local(path=None,
+                                                         config=triton_config,
+                                                         gpus=[],
+                                                         log_path="")
+        logger.warning(
+            "Model Analyzer currently does not support setting custom Triton server "
```

Comment:
Do we display the first sentence even if the user has not set any Triton server flags?